run.sh

```diff
@@ -14,5 +14,5 @@ migrate)
     bundle exec rake db:schema:load
     ;;
 esac
-bundle exec import:all assets:precompile
+RAILS_ENV=production bundle exec import:all assets:precompile
```

Comment:
We don't need `test` or `development` environments: they're only used locally and by our continuous integration service. *Everything* on production and pre-production – scripts, queue consumers, and web servers – should be run with `RAILS_ENV` set to `production`.